test/scenarios/applicationmonitoring/dataingest.go

```diff
@@ -43,42 +39,49 @@ type metadata struct {
 }
 
 func dataIngest(t *testing.T) features.Feature {
-	dataIngestFeature := features.New("data-ingest")
-	tenantSecret, err := secrets.DefaultSingleTenant(afero.NewOsFs())
-	dataIngestDynakube := dynakube.NewBuilder().
+	builder := features.New("data-ingest")
+	secretConfig := tenant.GetSingleTenantSecret(t)
+	testDynakube := dynakube.NewBuilder().
 		WithDefaultObjectMeta().
-		ApiUrl(tenantSecret.ApiUrl).
-		ApplicationMonitoring(&v1beta1.ApplicationMonitoringSpec{
+		ApiUrl(secretConfig.ApiUrl).
+		ApplicationMonitoring(&dynatracev1beta1.ApplicationMonitoringSpec{
 			UseCSIDriver: address.Of(false),
 		}).Build()
 
-	require.NoError(t, err)
+	sampleDeployment := sampleapps.NewSampleDeployment(t, testDynakube)
```

Comment:
why do we create a deployment and a pod at the same time? Shouldnt the deployment handle this alone without the need to create a standalone pod? or is this intended to check for pods, that does not have owner references?

Reply:
thats what my 2nd thought was, thanks